spec/features/page_feature_spec.rb

```diff
@@ -58,22 +56,15 @@ module Alchemy
           end
 
           it "should redirect to public child" do
-            visit "/#{default_language.code}/not-public"
-            expect(page.current_path).to eq("/#{default_language.code}/public-child")
-          end
-
-          context "and url has no language code" do
-            it "should redirect to url of public child with language code of default language" do
-              visit '/not-public'
-              expect(page.current_path).to eq("/#{default_language.code}/public-child")
-            end
+            visit "/not-public"
```

Comment:
Also here, only omit the locale prefix, if the `page.language.code` is the `default_locale`, otherwise it needs to be prepended.